test/format/failure_configs/CONFIG.WT-9772

```diff
@@ -0,0 +1,212 @@
+############################################
```

Comment:
Thank you! I looked at this PR specifically because I wanted to make sure the CONFIG was saved. I'm glad you thought so too!

Reply:
Yes, I also find that this config is good to have in our testing as it found the corner case of RTS not testing transaction ids of an update during recovery.